3rdparty/libconfig/scanctx.c

```diff
@@ -114,7 +114,17 @@ FILE *scanctx_push_include(struct scan_context *ctx, void *buffer,
     strcat(full_file, file);
   }
 
-  fp = fopen(full_file ? full_file : file, "rt");
+  if(full_file)
+  {
```

Comment:
On same line as previous if pls.

Reply:
no. this is part of library. and here used non hercules style
i used same style as in lib